### ISSUE TITLE
Hunter - explosive trap

### ISSUE THREAD

**Issue by kauski** [dalaranwow/dalaran-wow#4427]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who) Hunter explosive trap, triggers but no dmg is done upon trigger and no dmg "aoe" area on ground is not happenin

**Expected behaviour**: Tell us what should happen instead Enemy target triggers explosive trap, takes DMG and leaves a aoe dmg area on the ground.

**Steps to reproduce the problem**:

1. play hunter and make explosive trap
2.  Was survival specced, trap mastery 3 talent, also with entrapment 3 and trap mastery 3 talent
3.  Havent tested without those talents and this bug occured after 13.2.2017 server restart, it did work normally before that.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
Well duh, everybody knows that explosive trap should deal dmg states it in the tooltip anyways here http://www.wowhead.com/spell=191433/explosive-trap

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills** Tryed atleast in sholazar basin, not workin on any mob there, didnt work wintergrasp in pvp neither, havent tested elsewhere.

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary** Dont think screenshots are necessary for this case.

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/1196. The fix for it must be tested on PTR first
